Loadouts/blumsv_brit_l85_ddpm.hpp

```diff
@@ -0,0 +1,318 @@
+// Author: BWMF
+// Description: US: HK416 - Marine Pattern Desert
+
+#include "undef.hpp" // Reset defines
+
+// Rifle
+#define RIFLE "CUP_arifle_L85A2_G","CUP_arifle_L85A2_NG"
+#define RIFLE_MAG "30Rnd_556x45_Stanag_red:8","30Rnd_556x45_Stanag_Tracer_Red:2"
+#define RIFLE_ATTACHMENTS "CUP_acc_ANPEQ_2"
+#define ALT_OPTICS STANAG_OPTICS
+// GL Rifle
+#define GLRIFLE "CUP_arifle_L85A2_GL"
+#define GLRIFLE_MAG RIFLE_MAG
+#define GLRIFLE_MAG_SMOKE "1Rnd_Smoke_Grenade_shell:2","1Rnd_SmokeRed_Grenade_shell:2"
+#define GLRIFLE_MAG_HE "1Rnd_HE_Grenade_shell:5"
+// Carbine
+#define CARBINE RIFLE
+#define CARBINE_MAG RIFLE_MAG
+// AR
+#define AR "CUP_lmg_L110A1"
+#define AR_MAG "CUP_100Rnd_TE4_Red_Tracer_556x45_M249:4"
+// AT
+#define AT "CUP_launch_M136"
+#define AT_MAG "ACE_PreloadedMissileDummy_CUP:1"
+// MMG
+#define MMG "CUP_lmg_L7A2"
+#define MMG_MAG "CUP_100Rnd_TE4_LRT4_Red_Tracer_762x51_Belt_M:5"
+// MAT
+#define MAT "CUP_launch_Mk153Mod0"
+#define MAT_MAG "CUP_SMAW_HEAA_M:3","CUP_SMAW_HEDP_M:1","CUP_SMAW_Spotting:3"
+#define MAT_MAG2 "CUP_SMAW_HEAA_M:2","CUP_SMAW_HEDP_M:1","CUP_SMAW_Spotting:2"
+#define MAT_OPTIC "CUP_optic_SMAW_Scope"
+//HMG
+#define HMG "ace_cswCompatCUP_cup_m2"
+#define HMG_TRI "ace_csw_m3CarryTripod"
+#define HMG_TRI_LOW "ace_csw_m3CarryTripodLow"
+#define HMG_MAG "ace_csw_100Rnd_127x99_mag:3"
+//HAT
+#define HAT "ace_cswCompatCUP_cup_tow"
+#define HAT_TRI "ace_csw_m220CarryTripod"
+#define HAT_TRI_LOW ""
```

Comment:
@TheCandianVendingMachine Is this right?

Reply:
If you're asking if they're the right items, yes https://puu.sh/FJ7w7/ca11f628fe.jpg